src/consensus/params.h

```diff
@@ -17,13 +17,21 @@ enum DeploymentPos
 {
     DEPLOYMENT_TESTDUMMY,
     DEPLOYMENT_ASSETS, // Deployment of RIP2
-    DEPLOYMENT_SCRYPT2, 
     // DEPLOYMENT_CSV, // Deployment of BIP68, BIP112, and BIP113.
 //    DEPLOYMENT_SEGWIT, // Deployment of BIP141, BIP143, and BIP147.
     // NOTE: Also add new deployments to VersionBitsDeploymentInfo in versionbits.cpp
     MAX_VERSION_BITS_DEPLOYMENTS
 };
 
+enum NetworkPeriod : int
+{
+    NETWORK_PERIOD_X16R = 0,
+    NETWORK_PERIOD_MAINTANCE,
+    NETWORK_PERIOD_SCRYPT2,
+
+    MAX_NETWORK_PERIOD
```

Comment:
I propose to rename with CURRENT_NETWORK_PERIOD

NETWORK_PERIOD_X16R = 0,
...

CURRENT_NETWORK_PERIOD = NETWORK_PERIOD_SCRYPT2

Reply:
+